kubetest2-gke/deployer/build/gke_make.go

```diff
@@ -44,7 +44,7 @@ const (
 )
 
 var (
-	gkeMinorVersionRegex = regexp.MustCompile(`^v(\\d\\.\\d+).*$`)
+	gkeMinorVersionRegex = regexp.MustCompile(`^v(\d\.\d+).*$`)
```

Comment:
let's reuse the regex at
https://github.com/kubernetes-sigs/kubetest2/blob/master/kubetest2-gke/deployer/version.go#L43

we already have a few unit tests for that

Reply:
The capture group in that regex is a bit tricky to get information from

Reply:
ACK, let's keep it simple here then.